src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,107 @@
+* {
+  margin: 0;
+}
+
+body {
+  font-family: $bodyFont, sans-serif;
+}
+
+.product-card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  width: 200px;
+  height: 408px;
```

Comment:
Don't hardcode height for the parent flex element

If you have the correct styles for children cards will have the same height without this rule